service-manual/making-software/information-security.md

```diff
@@ -26,15 +26,14 @@ taken to guarantee its security.
 
 Information security is a topic both broad and deep, drawing from fields ranging from economics and psychology through to mathematics and probability.
 
-This document doesn’t provide a thorough review of the field. Instead, it aims to:
+This document doesn't provide a thorough review of the field. Instead, it aims to:
```

Comment:
Any reason why `’` has been changed to `'` throughout this commit? The rest of the service manual used typographic quote marks.